### DISCUSSION TITLE
Tiny elements / Fractional scaling support

### DISCUSSION THREAD

**Discussion by konceptosociala** [emilk/egui#3052] [Q&A]
I use Ubuntu fractional scaling of the interface for my laptop, as its screen is pretty small, but the resolution is full HD, so its interface is very little, however decreasing the resolution or increasing text size just makes it look ugly. But even with turned fractional scaling on, my `egui` (egui + ash + winit) app's elements are still very small and unclear, and as I said before increasing font size wouldn't deal with the issue.

What should I do to support every kind of monitors? (as the app on other my PCs looks regular)

![Ekrankopio de 2023-06-03 13-30-09](https://github.com/emilk/egui/assets/43478202/bebc93af-c2a8-4940-8968-f7ff95467f7c)

**Comment by JumpyLionnn** [comment]
You can try to use this to set global scaling
```rust
ctx.set_pixels_per_point(number);
```
You can try to calculate the amount of pixels based on the window size
